mkdocs.yml

```
@@ -5,16 +5,20 @@ nav:
     - v3:
       - "Console Helper": v3/console-helper.md
       - Migration: v3/migration.md
+    - v4:
+        - "Console Helper": v4/console-helper.md
+        - Migration: v4/migration.md
```

An minor version may also need descriptions, therefore:

```suggestion
        - Migration:
           - "Migration from Version 3 to 4": v4/migration/v3-to-v4.md
```

- Reference: https://github.com/laminas/documentation/issues/13
- Example: https://docs.laminas.dev/laminas-form/v3/migration/v2-to-v3/